src/reporting/lambda/custom-resource/quicksight/index.ts

```
@@ -1102,7 +1103,7 @@ const createOrUpdateRefreshSchedule = async (quickSight: QuickSight, commonParam
         logger.info('RefreshProperties exist, skip put operation.');
       } else {
         logger.error(`Put QuickSight refresh properties failed due to: ${(err as Error).message}`);
-        throw err;
+        // throw err;
```

don't ignore the exception